installers/mac/pkg/build.gradle

```diff
@@ -48,9 +48,7 @@ task inflatePkgTemplate {
         copy {
             from('templates/CorrettoPkg.pkgproj.template') {
                 rename { file -> file.replace('.template', '') }
-                project.version.each {
-                    prop -> filter(org.apache.tools.ant.filters.ReplaceTokens, tokens: [ (prop.key): String.valueOf(prop.value)])
-                }
+                filter(org.apache.tools.ant.filters.ReplaceTokens, tokens: ["full": project.version["full"], "major": project.version["major"], "arch": correttoArch, "macos-arch": (correttoArch == "aarch64" ? "arm64" : "x86_64")])
```

Comment:
Can macos-arch just be nativeArch from toplevel build.gradle?

Reply:
no - we need `nativeArch` to follow existing convention and be `aarch64` for other places in our build logic. But for this substitution specifically it should be `arm64` instead.